PencilCase/PencilCase/scripts/Pages/DealPageBase.ts

```diff
@@ -8,6 +8,19 @@ import { Order } from '../Models/Order';
 import * as Utils from '../Utils';
 import { OrderTypes } from '../Models/Order';
 
+/*
+* Extend ko functionality
+* https://stackoverflow.com/questions/12822954/get-previous-value-of-an-observable-in-subscribe-of-same-observable
+*/
+ko.subscribable.fn.subscribeChanged = function (callback, dataContext) {
```

Comment:
Need to the old value and new value at the same time to re-calculate the invalid order count. If changed from invalid to valid, invalid order account decrease 1. If changed from valid to invalid, invalid order account increase 1,